<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(22, 45, 68);
border-radius:10px;</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Upload Image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="upload_img">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color: rgb(255, 255, 255);
 background-color: rgb(14, 139, 255);
border-radius:10px;
font: 10pt &quot;Arial Rounded MT Bold&quot;;
}

QPushButton:hover{
color: rgb(255, 255, 255);
	background-color: rgb(0, 170, 255);
font: 10pt &quot;Arial Rounded MT Bold&quot;;
}
</string>
      </property>
      <property name="text">
       <string>Upload</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../Qt Desiger/Icon.qrc"/>
 </resources>
 <connections/>
</ui>
